mass-deployer/pkg/mass-deployer/types.go

```diff
@@ -0,0 +1,44 @@
+package deployer
```

Comment:
also please check this package to integrate validators https://pkg.go.dev/gopkg.in/validator.v2